src/components/common/ErrorView/styles.js

```diff
@@ -3,14 +3,14 @@ import { ERROR } from 'constants/colors';
 
 const styles = StyleSheet.create({
   container: {
+    textAlign: 'center',
+    alignItems: 'center',
     justifyContent: 'center',
-    padding: 12,
   },
   error: {
     color: ERROR,
-    fontSize: 14,
-    fontWeight: '500',
-    marginBottom: 8,
+    fontSize: 10,
```

Comment:
What do you think about creating a typography file to group all the font styles?